grpc-ballerina/tests/44_route_guide.proto

```diff
@@ -0,0 +1,111 @@
+// Copyright 2015 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Shall we add a comment here, mentioning the reference

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-grpc/pull/275/commits/098ffaf1db5e6b65a2721e60a4f97c0e59d78605